pkg/rpc/server/server_test.go

```diff
@@ -0,0 +1,162 @@
+package server
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sync"
+	"testing"
+	"time"
+
+	"golang.org/x/xerrors"
+
```

Comment:
Remove extra new lines here.

Reply:
I fixed it.
https://github.com/aquasecurity/trivy/pull/307/commits/c09066f7992a7107c75769e399cab1abc3f6159f